src/media/js/site/containers/devAgreement.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
can we add module-level multi-line JS comment to devAgreement.js and tos.js? sometimes get confused on which is used where since they seem similar